docs/content/users/configuration/experimental.md

```diff
@@ -26,3 +26,43 @@ Rancher Desktop integration currently has no automated testing for DDEV integrat
 * Rancher Desktop does not provide bind mounts, so use `ddev config global --no-bind-mounts` which also turns on Mutagen.
 * Use a non-`ddev.site` name, `ddev config --additional-fqdns=rancher` for example, because the resolution of `*.ddev.site` seems to make it not work.
 * Rancher Desktop does not seem to currently work with `mkcert` and `https`, so turn those off with `mkcert -uninstall && rm -r "$(mkcert -CAROOT)"`. This does no harm and can be undone with just `mkcert -install` again.
+
+## Traefik Router
+
+The job of the `ddev-router` is to receive most HTTP or HTTPS traffic, like requests to `*.ddev.site`, and deliver it to the correct project's web container, see [Container Architecture](../basics/architecture.md#container-architecture). The router as of DDEV v1.21.3 was always a forked, poorly documented nginx reverse proxy container.
+
+The very popular [Traefik Proxy](https://traefik.io/traefik/) will replace the current `ddev-router` in the future, but is available as an experimental configuration today. Feedback is welcome. To enable it:
```

Comment:
```suggestion
Until DDEV v1.21.3, `ddev-router` was been based on a forked, poorly-documented nginx reverse proxy. A future release will replace this with the popular [Traefik Proxy](https://traefik.io/traefik/), available now as an experimental feature. Run the following to enable it:
```

Reply:
Whoops! I misread that as `ddev-router` starting with its current forked nginx proxy in v1.21.3.

Your edit fixes my mistake, but also seems to suggest that in DDEV v1.21.3 `ddev-router` is no longer based on that nginx fork—but it still is by default if I’m understanding correctly. I edited my suggestion above to clarify that 1.21.3 adds an opt-in Traefik option that will become the default in some future release:

> The default `ddev-router` is based on a forked, poorly-documented nginx reverse proxy. A future release will replace this with the popular [Traefik Proxy](https://traefik.io/traefik/), available in DDEV v1.21.3 as an experimental feature. Run the following to enable it:

Reply:
...and one more revision to try and clarify that 1.21.3 adds a new option for the first time. It’s weird to start the paragraph calling it a “default” when it’s been the _only_ option before this new introduction:

> `ddev-router` has been based on a forked, poorly-documented nginx reverse proxy. DDEV v1.21.3 adds a new router based on the popular [Traefik Proxy](https://traefik.io/traefik/), available as an experimental feature until it becomes the default in a future release. Run the following to enable it:

Updated my suggestion above and I promise I’ll leave it alone now.